docs/help.css

```
@@ -149,11 +149,11 @@ input[type='checkbox'] {
     border-left: 4px solid currentColor;
     vertical-align: middle;
     margin-right: .7rem;
-    transform: translateY(-2px);
+    transform: rotate(90deg) translateX(-3px);
 }
 
 .toggle:checked + .lbl-toggle::before {
-    transform: rotate(90deg) translateX(-3px);
+    transform: rotate(0deg) translateY(-3px);
```

Maybe you can write this to be the same as the original appearance and code style.

The original code of the 152nd line:

```diff
-     transform: translateY(-2px);
```

```suggestion
    transform: translateY(-2px);
```